### PR TITLE
Allows usage of ignoreClassMethods from istanbul

### PR DESCRIPTION
**Pull request by katerberg**
This allows ignoring of class methods of a certain name as a whole rather than having to manually ignore each instance. It is the companion to https://github.com/istanbuljs/istanbuljs/pull/127

### CODE REVIEW

**Comment by katerberg** [comment]
@bcoe I think I got all of it in now! Let me know what you think.

File: test/nyc-bin.js [hunk 1]
```diff
@@ -71,6 +71,30 @@ describe('the nyc cli', function () {
     })
   })
 
+  describe('--ignore-class-method', function () {
```
Comment (bcoe) [inline test/nyc-bin.js]: awesome, thank you for the test 👍

File: test/nyc-bin.js [hunk 2]
```diff
@@ -71,6 +71,30 @@ describe('the nyc cli', function () {
...
+    if (parseInt(process.versions.node.split('.')[0]) < 4) return
```
Comment (bcoe) [inline test/nyc-bin.js]: I don't think this check should be needed; we dropped Node <4 support in CI a few months ago.

File: README.md [hunk 3]
```diff
@@ -272,6 +272,7 @@ Any configuration options that can be set via the command line can also be speci
     "exclude": [
       "src/**/*.spec.js"
     ],
+    "ignore-class-method: "methodToIgnore",
```
Comment (bcoe) [inline README.md]: theres a missing closing `"` on `ignore-class-method`.

**Comment by katerberg** [comment]
Updated @bcoe! Let me know if you'd like anything else changed.

**Comment by bcoe** [comment]
@katerberg your work is released, please give the latest version of nyc a shot.

**Comment by katerberg** [comment]
Worked beautifully! Thanks for merging it. It's nice to have our project back on mainline :D

**Pull request merged**